posthog/api/test/test_plugin.py

```diff
@@ -959,6 +959,59 @@ def test_plugin_config_attachment(self, mock_get, mock_reload):
         self.assertEqual(response.json()["config"], {"bar": "moop"})
         self.assertEqual(PluginAttachment.objects.count(), 0)
 
+        response = self.client.get("/api/organizations/@current/plugins/activity")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        changes = response.json()["results"]
+
+        self.assertEqual(len(changes), 5)
+
+        for i in (0, 1, 2, 3):
```

Comment:
This is a pretty huge test, now spanning nearly 130 lines. We also need to be doing some 'programming' and matching up these later assertions with the earlier requests is non-trivial.

In general all of the above is a sign that the test is doing way too much. Would it make sense to put _this_ part of it into a new test instead or split the existing test?

Reply:
Agreed - just having the test file be 1300 lines and plugin.py ~700 was a bit of a maze to navigate. I definitely think this can be split up / refactored.